cocaine12/worker.go

```diff
@@ -260,7 +224,7 @@ func (w *Worker) isStopped() bool {
 	return false
 }
 
-func (w *Worker) loop() error {
+func (w *Worker) Loop() error {
```

Comment:
We do not expect to use EventHandlers in our code. We expect to use SetHandler. So we either need a Run(void)+SetHandler, or Run(Handler).
I can change it to Run(Handler) and the usage of EventHandlers will be Run(handlers.Call)